packages/elastic-sip-trunking/src/models/v1/sip-endpoint/sip-endpoint.ts

```diff
@@ -21,5 +21,5 @@ export interface SipEndpoint {
   /** The date and time that the SIP endpoint was created. */
   createTime?: Date;
   /** The date and time that the SIP endpoint was last modified. */
-  updateTime?: Date;
+  updateTime?: Date | null;
```

Comment:
According to PR's comment:

> The updateTime property will always be returned

Does it have sense to let `updateTime` as optional ?

Reply:
🤷‍♂️
It's the same question for the property `createTime` defined above: one can assume that a SIP Endpoint object can be returned only after it has been created first but the documentation specifies it as optional still.
And there are also properties with default values such as `port`, `transport` and `enabled`. 
`id` and `sipTrunkId` are also noted as optional but in reality, they will also be always defined, the `id` for obvious reasons and the `sipTrunkId` because it's a mandatory parameter in order to create a SIP Endpoint